### PR TITLE
LLM: add peak diff

### PR DESCRIPTION
**Pull request by WeiguangHan**
Add peak diff.

### CODE REVIEW

File: python/llm/test/benchmark/csv_to_html.py [hunk 1]
```diff
@@ -30,8 +30,23 @@ def highlight_vals(val, max=3.0):
     else:
         return ''
 
-def is_diffs_within_normal_range(diff1, diff2, threshold=5.0):
-    return not any(diff < (-threshold) for diff in diff1 + diff2)
+def is_diffs_within_normal_range(diff1, diff2, best_diff1, best_diff2, threshold=5.0):
```
Comment (Oscilloscope98) [inline python/llm/test/benchmark/csv_to_html.py:33]: Has discussed with @liu-shaojun , there is no need to add best_diff1 and best_diff2 into this function to judge whether they are in the normal range :)

Comment (Oscilloscope98) [reply python/llm/test/benchmark/csv_to_html.py:33]: Besides, in the main branch
image
This error has been fixed. Please rebase this PR. Thank you :)

Comment (WeiguangHan) [reply python/llm/test/benchmark/csv_to_html.py]: Sure.

File: python/llm/test/benchmark/csv_to_html.py [hunk 2]
```diff
@@ -106,11 +146,17 @@ def main():
         latest_csv.insert(loc=5,column='last2',value=last2)
         latest_csv.insert(loc=6,column='diff2(%)',value=diff2)
 
-        diffs_within_normal_range = is_diffs_within_normal_range(diff1, diff2, highlight_threshold)
+        latest_csv.insert(loc=7,column='best 1',value=best_last1)
+        latest_csv.insert(loc=8,column='best diff1(%)',value=best_diff1)
+        latest_csv.insert(loc=9,column='best 2',value=best_last2)
+        latest_csv.insert(loc=10,column='best diff2(%)',value=best_diff2)
+
+        diffs_within_normal_range = is_diffs_within_normal_range(diff1, diff2, best_diff1, best_diff2, highlight_threshold)
```
Comment (Oscilloscope98) [inline python/llm/test/benchmark/csv_to_html.py:154]: There is no need to add best_diff1 and best_diff2 in this function argument :)

Comment (WeiguangHan) [reply python/llm/test/benchmark/csv_to_html.py:154]: Thanks for reminding！

**Review by Oscilloscope98 — APPROVED** [review]
LGTM

**Pull request merged**